Procfile

```diff
@@ -1 +1,2 @@
 web: gunicorn authors.wsgi
+release: python manage.py migrate --noinput
```

Comment:
@felixkiryowa this change is no longer necessary.

Reply:
I have fixed it